packages/mobile/android/app/build.gradle

```diff
@@ -223,7 +223,6 @@ android {
 dependencies {
     implementation project(':react-native-install-referrer')
     implementation project(':react-native-send-intent')
-    // implementation project(':instabug-reactnative')
     // implementation project(':react-native-screens')
```

Comment:
unrelated: do you know why `react-native-screesn` was removed?

Reply:
@cmcewen do you remember why `react-native-screens` was disabled?